### ISSUE TITLE
MPI installed ?

### ISSUE THREAD

**Issue by simonweppe** [tomsail/telemac-conda#23]
Hi @tomsail,

Looks like Im not able to run in parallel, is MPI meant to be functional ? 

For example, if I try : 
```
cd /examples/artemis/creocean
artemis.py art_creocean_2.cas
```
It will complain about MPI.

It works if I change the solver back to 8, as in `/examples/artemis/beach/art_plage.cas` for example (this one works fine).

**Comment by tomsail** [comment]
Hi Simon, ARTEMIS in parallel requires the installation of MUMPS. 
For the moment I haven't succeeded to install it.

However, the [conda package](https://github.com/tomsail/telemac-mascaret-feedstock) developped by @nicogodet for windows has it. 

I am currently trying to match his setup (as he is just mirroring the files from the official repo), and I'll see also I can get MUMPS to work in the process.

**Comment by nicogodet** [comment]
Important note: mine is windows only

**Comment by simonweppe** [comment]
Ok noted thanks for the quick replies, and your work. Making life a lot easier when trying to play with Telemac !

**Comment by tomsail** [comment]
Hi @simonweppe, this has been fixed now. 
The package has been renamed to `opentelemac`. 

So to have ARTEMIS working in parallel: 
   ```console
   mamba create -n telemac -c tomsail opentelemac
   mamba activate telemac
   cd $EXAMPLES_PATH/examples/artemis/creocean
   artemis.py art_creocean.cas --ncsize 4
   ```
Note the `--ncsize 4`: It has to be multi-threaded, if you don't specify ncsize it won't work.

I haved updated and added all info on README.md 

There is also the windows version available thanks to @nicogodet's great work!
